### ISSUE TITLE
ARFF parser fails when att value contains comma

### ISSUE THREAD

**Issue by janvanrijn** [openml/openml-python#404]
e.g., dataset 40725

```
@relation predictit

@attribute date {2017-04-21}
@attribute hour numeric
@attribute minute numeric
@attribute ticker {MINWAGE.2017,INDIV.TAXCUT.2017,CORP.TAXCUT.2017,INDMANDATE.ACA.2017,EMPMANDATE.ACA.2017,DEVICETAX.ACA.2017,SUBSIDY.ACA.2017,PREEXISTING.ACA.2017,'ENACT.NOGOPMAJ,2017',ABOLISHEC.AMENDMENT.123117,NRTW.2017,CARRIEDINTEREST.2017,TPP.123117,ISRAELPM.CONGRESS.2017,SANCFUNDING.123117,OBAMATESTIFY.2017,NUCLEAR.HC.2017,RYAN.SPEAKER.063017,RUSSIA.SELECTCMTE.063017,RICETESTIFY.2017,NUNES.INTEL.063017,180.CONGAPPR.042417,175.CONGAPPR.042417,170.CONGAPPR.042417,165.CONGAPPR.042417,164.CONGAPPR.042417,AHCA.HOUSE.042817,KENNEDY.NEXTVACANCY.SCOTUS.120916,GINSBURG.NEXTVACANCY.SCOTUS.120916,THOMAS.NEXTVACANCY.SCOTUS.120916,BREYER.NEXTVACANCY.SCOTUS.120916,ALITO.NEXTVACANCY.SCOTUS.120916,ROBERTS.NEXTVACANCY.SCOTUS.120916,KAGAN.NEXTVACANCY.SCOTUS.120916,SOTO.NEXTVACANCY.SCOTUS.120916,VACANCY.SCOTUS.123117,SCOTUS.EMOLUMENTS.2017,SCOTUS.TRAVELEO.2017,SCOTUS.NCAANJ.2017,YELLEN.063017,CHINA.MANIPULATOR.063017,150.DEBTCHANGE.012018,125.DEBTCHANGE.012018,100.DEBTCHANGE.012018,75.DEBTCHANGE.012018,50.DEBTCHANGE.012018,25.DEBTCHANGE.012018,0.DEBTCHANGE.012018,DEC.DEBTCHANGE.012018,COMEY.063017,CLINTON.FEDCHARGES.2017,CUOMO.123117,CHRISTIE.123117,CORDRAY.123117,TRUMP.ZERVOS.2017,OBAMAHOLIDAY.2017,PALIN.AMBCANADA.123117,SNOWDEN.2017,GOTT.FDA.063017,ONEI.FDA.063017,OSTR.FDA.063017,GULF.FDA.063017,SRIN.FDA.063017,FLYNN.FEDCHARGES.2017,TRUMP.WHCD.042917,PENCE.WHCD.042917,LA.OLYMPICS2024,SESSIONS.ALSANCTION.2017,SHUTDOWN.050117,FLYNN.IMMUNITY.043017,1500.BHOSPEECH,1250.BHOSPEECH,1000.BHOSPEECH,750.BHOSPEECH,500.BHOSPEECH,499.BHOSPEECH,370.DIRECTION.042417,365.DIRECTION.042417,360.DIRECTION.042417,355.DIRECTION.042417,354.DIRECTION.042417,SESSIONS.AG.043017,PERD.AGSEC.043017,RAST.AGSEC.043017,CONN.AGSEC.043017,45.IMPEACH.042917,PUTIN.TRUMP100D,RUSSIA.TRUMP100D,EMBASSY.JERUSALEM.2017,PARISTREATY.2017,TAIWAN.TRUMP.2017,QEII.TRUMP100D,TRUMP.PRES.123117,VETO.TRUMP100D,SPICER.WHPS.123117,PENANIETO.TRUMP100D,650.TRUMPAPPR.043017,600.TRUMPAPPR.043017,550.TRUMPAPPR.043017,500.TRUMPAPPR.043017,450.TRUMPAPPR.043017,400.TRUMPAPPR.043017,350.TRUMPAPPR.043017,349.TRUMPAPPR.043017,TRUMP.PRES.123118,IMPEACH.TRUMP.123117,550.TRUMPAPPR.053117,500.TRUMPAPPR.053117,450.TRUMPAPPR.053117,400.TRUMPAPPR.053117,350.TRUMPAPPR.053117,349.TRUMPAPPR.053117,SPICER.WHPS.043017,PRIEBUS.WHCOS.APR2017,475.TRUMPAPPR.063017,450.TRUMPAPPR.063017,425.TRUMPAPPR.063017,400.TRUMPAPPR.063017,375.TRUMPAPPR.063017,374.TRUMPAPPR.063017,434.TRUMPAPPR.042117,430.TRUMPAPPR.042117,426.TRUMPAPPR.042117,422.TRUMPAPPR.042117,418.TRUMPAPPR.042117,414.TRUMPAPPR.042117,413.TRUMPAPPR.042117,PENCE.TRUMPTWEET.042517,4.POTUSTWEETS.042517,5.POTUSTWEETS.042517,10.POTUSTWEETS.042517,15.POTUSTWEETS.042517,20.POTUSTWEETS.042517,25.POTUSTWEETS.042517,19.TRUMPTWEETS.042617,20.TRUMPTWEETS.042617,25.TRUMPTWEETS.042617,30.TRUMPTWEETS.042617,35.TRUMPTWEETS.042617,40.TRUMPTWEETS.042617,SHUTDOWN.TRUMPTWEET.042617,SPICER.042117,14.POTUS.042617,15.POTUS.042617,20.POTUS.042617,25.POTUS.042617,30.POTUS.042617,35.POTUS.042617,40.POTUS.042617,45.POTUS.042617,GALLUP.042817,434.TA.042817,430.TA.042817,426.TA.042817,422.TA.042817,418.TA.042817,414.TA.042817,413.TA.042817,39.VPTWEET.042817,40.VPTWEET.042817,50.VPTWEET.042817,60.VPTWEET.042817,70.VPTWEET.042817,80.VPTWEET.042817,90.VPTWEET.042817,100.VPTWEET.042817}
@attribute buy_yes numeric
@attribute but_no numeric
@attribute val numeric

@data
2017-04-21,12,58,MINWAGE.2017,0.13,0.92,6
2017-04-21,12,58,INDIV.TAXCUT.2017,0.4,0.62,6
2017-04-21,12,58,CORP.TAXCUT.2017,0.63,0.48,6
2017-04-21,12,58,INDMANDATE.ACA.2017,0.37,0.66,6
2017-04-21,12,58,EMPMANDATE.ACA.2017,0.37,0.66,6
2017-04-21,12,58,DEVICETAX.ACA.2017,0.38,0.65,6
2017-04-21,12,58,SUBSIDY.ACA.2017,0.29,0.76,6
2017-04-21,12,58,PREEXISTING.ACA.2017,0.14,0.9,6
2017-04-21,12,58,'ENACT.NOGOPMAJ,2017',0.17,0.87,6
2017-04-21,12,58,ABOLISHEC.AMENDMENT.123117,0.02,0.99,6
```

**Comment by janvanrijn** [comment]
In this case the ticket attribute ENACT.NOGOPMAJ,2017 (fails on line 20)

Although this is a liac-arff problem, I opened it here as LIAC arff does not seem to be supported anymore

**Comment by mfeurer** [comment]
Should be fixed by #427.
